data/beta/field/statistical-geography.yaml

```diff
@@ -3,4 +3,4 @@ datatype: string
 field: statistical-geography
 phase: beta
 register: statistical-geography
-text: The entity (record) for geographies that are used in official statistics.
\ No newline at end of file
+text: The unique code of the entity for geographies that are used in official statistics.
```

Comment:
'the unique *for* the entity'...?

Reply:
After discussion with Duncan, let's go with: The unique code for a geography used in official statistics. A geography is a way to divide a larger area into smaller areas.

Reply:
Updated with new description @trvllngmn @mpwoods